### ISSUE TITLE
Pretty_print() indentation issue

### ISSUE THREAD

**Issue by pedro-mendonca** [pbiron/updates-api-inspector#7]
Hi,

The function pretty_print() is checking for `)` and `),` lines to reduce indentation.
The thing is that some lines that should also be checked are `))` and `)),` so the indentation is fixed.

As you were careful to output the data according WPCS (e.g.: alignment of `=>` items), I suggest adding a linebreak before the array in `__set_state( array(`.

So, the solution is just to add a linebreak in both `(array(` and `))` strings and it's all set and fixed because you won't have anymore the `))` missing indentations! :)

Here is an example:
```
stdClass::__set_state(array(
	'updates' => array(
		stdClass::__set_state(array(
			'response' => 'development',
			'download' => 'https://wordpress.org/nightly-builds/wordpress-5.5-latest.zip',
			'locale' => 'en_US',
			'packages' => stdClass::__set_state(array(
				'full' => 'https://wordpress.org/nightly-builds/wordpress-5.5-latest.zip',
				'no_content' => '',
				'new_bundled' => '',
				'partial' => '',
				'rollback' => '',
				)),
				'current' => '5.5-RC2-48740',
				'version' => '5.5-RC2-48740',
				'php_version' => '5.6.20',
				'mysql_version' => '5.0',
				'new_bundled' => '5.3',
				'partial_version' => '',
				)),
				stdClass::__set_state(array(
					'response' => 'latest',
					'download' => 'https://downloads.wordpress.org/release/wordpress-5.4.2.zip',
					'locale' => 'en_US',
					'packages' => stdClass::__set_state(array(
						'full' => 'https://downloads.wordpress.org/release/wordpress-5.4.2.zip',
						'no_content' => 'https://downloads.wordpress.org/release/wordpress-5.4.2-no-content.zip',
						'new_bundled' => 'https://downloads.wordpress.org/release/wordpress-5.4.2-new-bundled.zip',
						'partial' => '',
						'rollback' => '',
						)),
						'current' => '5.4.2',
						'version' => '5.4.2',
						'php_version' => '5.6.20',
						'mysql_version' => '5.0',
						'new_bundled' => '5.3',
						'partial_version' => '',
						)),
						stdClass::__set_state(array(
							'response' => 'autoupdate',
							'download' => 'https://wordpress.org/nightly-builds/wordpress-5.5-latest.zip',
							'locale' => 'en_US',
							'packages' => stdClass::__set_state(array(
								'full'            => 'https://wordpress.org/nightly-builds/wordpress-5.5-latest.zip',
								'no_content'      => '',
								'new_bundled'     => '',
								'partial'         => '',
								'rollback'        => '',
								)),
								'current'         => '5.5-RC2-48740',
								'version'         => '5.5-RC2-48740',
								'php_version'     => '5.6.20',
								'mysql_version'   => '5.0',
								'new_bundled'     => '5.3',
								'partial_version' => '',
								'new_files'       => '1',
								)),
							),
							'last_checked' => 1596720174,
							'version_checked' => '5.5-RC2-48740',
							'translations' => array(),
							))
```

**Comment by pedro-mendonca** [comment]
`pretty_print()` output after fix #8:

```
stdClass::__set_state(
	array(
		'updates'         => array(
			stdClass::__set_state(
				array(
					'response'        => 'development',
					'download'        => 'https://wordpress.org/nightly-builds/wordpress-5.5-latest.zip',
					'locale'          => 'en_US',
					'packages'        => stdClass::__set_state(
						array(
							'full'        => 'https://wordpress.org/nightly-builds/wordpress-5.5-latest.zip',
							'no_content'  => '',
							'new_bundled' => '',
							'partial'     => '',
							'rollback'    => '',
						)
					),
					'current'         => '5.5-RC2-48740',
					'version'         => '5.5-RC2-48740',
					'php_version'     => '5.6.20',
					'mysql_version'   => '5.0',
					'new_bundled'     => '5.3',
					'partial_version' => '',
				)
			),
			stdClass::__set_state(
				array(
					'response'        => 'latest',
					'download'        => 'https://downloads.wordpress.org/release/wordpress-5.4.2.zip',
					'locale'          => 'en_US',
					'packages'        => stdClass::__set_state(
						array(
							'full'        => 'https://downloads.wordpress.org/release/wordpress-5.4.2.zip',
							'no_content'  => 'https://downloads.wordpress.org/release/wordpress-5.4.2-no-content.zip',
							'new_bundled' => 'https://downloads.wordpress.org/release/wordpress-5.4.2-new-bundled.zip',
							'partial'     => '',
							'rollback'    => '',
						)
					),
					'current'         => '5.4.2',
					'version'         => '5.4.2',
					'php_version'     => '5.6.20',
					'mysql_version'   => '5.0',
					'new_bundled'     => '5.3',
					'partial_version' => '',
				)
			),
			stdClass::__set_state(
				array(
					'response'        => 'autoupdate',
					'download'        => 'https://wordpress.org/nightly-builds/wordpress-5.5-latest.zip',
					'locale'          => 'en_US',
					'packages'        => stdClass::__set_state(
						array(
							'full'        => 'https://wordpress.org/nightly-builds/wordpress-5.5-latest.zip',
							'no_content'  => '',
							'new_bundled' => '',
							'partial'     => '',
							'rollback'    => '',
						)
					),
					'current'         => '5.5-RC2-48740',
					'version'         => '5.5-RC2-48740',
					'php_version'     => '5.6.20',
					'mysql_version'   => '5.0',
					'new_bundled'     => '5.3',
					'partial_version' => '',
					'new_files'       => '1',
				)
			),
		),
		'last_checked'    => 1596721681,
		'version_checked' => '5.5-RC2-48740',
		'translations'    => array(),
	)
)
```

**Comment by pbiron** [comment]
Thanx @pedro-mendonca .

PHP switched `var_export()` to output objects as `(object) array()` instead of `stdClass::__set_state(array(...`.  I thought it was quite a while ago so I didn't bother to account for the old representation in the pretty printer.

But looking at the [PHP manual](https://www.php.net/manual/en/function.var-export), I see it was only in 7.3 that the change was made, so it is probably worthwhile accounting for the old behavior.

I'll look at #8 when I get a chance.

**Comment by pedro-mendonca** [comment]
That was a close one, I'm testing on PHP 7.2.9 :)

**Comment by pbiron** [comment]
Fixed in #8.
